Cargo.toml

```diff
@@ -22,7 +22,8 @@ lazy_static = "1.2"
 log = "0.4.11"
 hex = "0.4.3"
 
-opencl3 = { version = "0.6", default-features = false, features = ["CL_VERSION_1_2"], optional = true }
+opencl3 = { version = "0.9.1", default-features = false, features = ["CL_VERSION_1_2"], optional = true }
+opencl-sys = "0.2.3"
```

Comment:
I think `opencl-sys` doesn't need to be a direct dependency any more, due to the upgrade of `opencl3` to 0.9.1.

Reply:
Good catch! fixed!